### PR TITLE
Added Aquila integration documentation to /apps.

### PR DESCRIPTION
**Pull request by ricardobayes**

### CODE REVIEW

**Comment by antmonnier** [comment]
Janaina will review the content, however I've just lowercased the .PNG extension of image names. Also I've noticed that `hubrise_ID.png` was not used.

File: content/apps/aquila/en/getting_started.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+Once the HubRise plugin has been activated, the connection to a HubRise Account can be set through the following process:
+
+1. Login to your Aquila back office.
+2. Select Plugins from the left navigation panel.
+3. Click Configure on the HubRise Module row.
+4. To allow HubRise to create new products on Aquila, set Only create new product to Yes.  To allow the HubRise connection to update existing products in Aquila, set Only create new product to No.
+5. Click Login to HubRise.
+6. Select the account and the catalog you wish to use, and press Allow. Currently only account-level connection is possible.
+
+![](../images/plugins_annotated.png)
```
[the remark this answers is not in the document]
Comment (ricardobayes) [reply content/apps/aquila/en/getting_started.md]: This is blocked until we get another test account from Nextsourcia with the error message on the top of the site.

File: content/apps/aquila/en/troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+1. Login to your HubRise account. This can be done from your Aquila back office by selecting **Settings > Add-ons**, then click **Go to your HubRise account**.
+2. If prompted, log into your HubRise user account.
+3. From HubRise, click **Data** > **Orders**. The new Order will be displayed.
+4. Click the new Order to verify the customer and order information is accurate.
+5. The **Logs** section will contain the detailed JSON records of the communication between Aquila and HubRise. For more information on how to read these logs, see [Understanding Logs in HubRise](/docs/hubrise-logs).
+
+## Verify POS codes for All Products
+
+To verify that all Aquila menu items have had a POS code assigned, select **Catalog > Products** menu from the left navigation panel and check if there is a code filled under "Code".
```
[the remark this answers is not in the document]
Comment (ricardobayes) [reply content/apps/aquila/en/troubleshooting.md]: Yes, that is one of the questions asked.
I removed most of this content as it's not relevant. Toppings are not supported in Aquila, nor deals.
For that reason, I will not add a **Verify Deals Mapping** section to the troubleshooting.

File: content/apps/aquila/en/troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+These steps require that you have already connected Aquila to HubRise. For more information on connecting Aquila to HubRise, see [Getting Started](/apps/aquila/getting-started/).
+
+Once the connection is complete, test orders generated from Aquila will be transmitted to HubRise. Follow this process to create a test order and verify it has been transmitted to HubRise.
+
+1. Login to your Aquila back office.
+2. Select **View your website** from the top of the page. This will open your Aquila site in another browser tab or window.
+3. Generate an order.
+4. Return to Aquila, and from the left navigation panel click **Orders**. The new order will be present.
+
```
[the remark this answers is not in the document]
Comment (ricardobayes) [reply content/apps/aquila/en/troubleshooting.md:34]: I have reviewed and modified this section according to your feedback.

File: content/apps/aquila/en/getting_started.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+
+## Enable the HubRise plugin
+To enable the HubRise plugin in Aquila, follow these steps:
+
+1. Login to your Aquila back office.
+2. Select Plugins from the left navigation panel.
+3. Verify the Module HubRise is set to Active.
+
+Once complete, Aquila will complete the process of activating the plugin.
```
[the remark this answers is not in the document]
Comment (ricardobayes) [reply content/apps/aquila/en/getting_started.md]: I have removed that misleading section.

File: content/apps/aquila/en/configuration.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+  - **Price & Availability**: The options made available to the customer, usually in the form of sizes. For example: an 8" pizza versus a 12” pizza.
+  - **Toppings**: Ingredients set at the product category or the product level.
+- **Deals**: Deals offer customers savings based on an order. For example, if they order a pizza for €9 and a soda for €2 separately, it would be €11. A combo deal might offer customers if they buy a pizza and soda together it would be €9.
+
+In some cases, the EPOS will also require codes to understand some of the Aquila functionalities. Always refer to your EPOS documentation on the HubRise website for details.
+
+- **Payment Methods**: How customers can pay for their orders.
+- **Service Types**: These define how orders are handled such as Delivery, Collection and Eat-In.
+- **Delivery charge**: Delivery charges might apply.
+- **IMPORTANT NOTE**: It is important that each POS code field available in Aquila be mapped with the EPOS product codes. If the POS code is missing, the EPOS will not be unable to correctly process the order.
```
[the remark this answers is not in the document]
Comment (ricardobayes) [reply content/apps/aquila/en/configuration.md]: In the new structure, these mappings (payment methods, service types, etc.) have their individual sections, I moved them there. For their content, I have asked for help from Aquila as it's not clear to me how to set or change the EPOS code of them.

File: content/apps/aquila/en/getting_started.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+
+## Connect Aquila to HubRise
+
+The connection between Aquila to HubRise can be made in one click either to an existing HubRise Account, or a new one.  The process below details how to create a new HubRise account when connecting your Aquila site.  For details on how to connect to an already existing HubRise Account, see the FAQ.
```
[the remark this answers is not in the document]
Comment (ricardobayes) [reply content/apps/aquila/en/getting_started.md]: I created a new section for this in the FAQ. For the new account/location I need more info from Aquila.

File: content/apps/aquila/en/getting_started.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
+---
+title: Getting Started
+position: 2
+layout: documentation
+meta:
+  title: Aquila Connection to HubRise Getting Started
+  description: Instructions on connecting Aquila with HubRise
+---
+
+Connecting Aquila to HubRise requires the HubRise plugin to be enabled and Aquila to be connected to HubRise.
```
[the remark this answers is not in the document]
Comment (ricardobayes) [reply content/apps/aquila/en/getting_started.md]: It is not clear to me so I asked it in a question to Aquila.

File: content/apps/aquila/en/getting_started.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+
+## Install the HubRise plugin
```
[the remark this answers is not in the document]
Comment (ricardobayes) [reply content/apps/aquila/en/getting_started.md]: I haved asked Aquila this one as well.

**Comment by janainawittner** [comment]
To merge.

**Pull request merged**